scripts/standby_node_health_check/standby_node_health_check.py

```diff
@@ -0,0 +1,401 @@
+        return False
+
+    @staticmethod
+    def _run_and_parse_ibstat():
+        retcode, stdout = StandbyNodeHealthChecker._run_command("ibstat")
+        if retcode != 0:
+            print("Cannot run ibstat")
+            return {}
+        lines = stdout.splitlines()
+        ca_info = {}
+        current_ca = None
+
+        for line in lines:
+            line = line.strip()
+            if line.startswith("CA '"):
+                current_ca = line.split("'")[1]
+                ca_info[current_ca] = {}
+            elif current_ca and line.startswith("State:"):
+                ca_info[current_ca]["State"] = line.split(":")[1].strip().lower()
+            elif current_ca and line.startswith("Physical state:"):
+                ca_info[current_ca]["Physical_state"] = (
+                    line.split(":")[1].strip().lower()
+                )
+
+        return ca_info
+
+    @staticmethod
+    def _run_and_parse_ip_link_show():
+        ret_code, ip_link_show_output = StandbyNodeHealthChecker._run_command(
+            "ip link show"
+        )
+        if ret_code != 0:
+            print("Failed to run ip link show")
+            return {}
+        interfaces = StandbyNodeHealthChecker._parse_ip_link_output(ip_link_show_output)
+        return interfaces
+
+    @staticmethod
+    def _parse_ip_link_output(ip_link_output: str):
+        interface_regex = re.compile(r"^(\d+): (\S+):.*state (\S+) .*")
+        link_regex = re.compile(r"^\s+link/(\S+)")
+
+        interfaces = {}
+        lines = ip_link_output.splitlines()
+        current_interface = None
+        state = None
+
+        for line in lines:
+            interface_match = interface_regex.match(line)
+            if interface_match:
+                current_interface = interface_match.group(2)
+                state = interface_match.group(3)
+            else:
+                link_match = link_regex.match(line)
+                if link_match and current_interface:
+                    link_type = link_match.group(1)
+                    if link_type == "ether":
+                        interfaces[current_interface] = state.lower()
+
+        return interfaces
+
+    def _check_eth_interfaces(self):
+        interfaces_status = StandbyNodeHealthChecker._run_and_parse_ip_link_show()
+        result = True
+        for interface in self._mngmnt_interfaces:
+            if interface not in interfaces_status:
+                print(f"{interface} is not in the list of ether interfaces")
+                result = False
+            elif interfaces_status[interface] != "up":
+                print(
+                    f"Interface {interface} is not active {interfaces_status[interface]}"
+                )
+                result = False
+        if result:
+            print("All given ether interfaces are active")
+        return result
+
+    @staticmethod
+    def _check_if_ha_is_enabled():
+        command = UFM_HA_CLUSTER_COMMAND.format("is-ha")
```

Comment:
Maybe it will be better to print some precheck message for each check - for example:
"Checking if UFM HA is enabled..."

Reply:
The output right now is long, I think this would be too much prints....